haystack/preview/components/file_converters/html.py

```diff
@@ -0,0 +1,72 @@
+import logging
+from typing import List, Optional, Dict, Any, Union
+from pathlib import Path
+
+from haystack.preview.lazy_imports import LazyImport
+from haystack.preview import Document, component, default_to_dict, default_from_dict
+
+with LazyImport("Run 'pip install boilerpy3'") as boilerpy3_import:
+    from boilerpy3 import extractors
```

Comment:
We need to add this dependency to the CI as well, so tests can run

Reply:
Ah it seems that `boilerpy3` is already present in the dependencies. Anyway, let's list it explicitly and add it to `haystack-ai` as well!

Reply:
Deal, was it like this https://github.com/deepset-ai/haystack-preview-package/pull/7 ?